File: check_file_exist/check_file_exist.pl
#!/usr/bin/perl

# perl check_file_exist.pl
# Perl中-e操作符用于检测文件是否存在
use strict;
use warnings;

my $file = "./a.txt";
if (-e $file) {
    print "$file exists.\n";
} else {
    print "$file does not exist.\n";
}

File: check_file_exist/check_file_exist.lua
local file_path = "./a.txt"
local file = io.open(file_path, "r")

-- todo verify
-- run cmd: lua check_file_exist.lua
if file then
    print(file_path .. " exists.")
    file:close()
else
    print(file_path .. " does not exist.")
end
